_component_design/tooltip.md

```diff
@@ -13,28 +13,292 @@ last-modified: 2017-08-17
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-text-conventions-1.svg){:width="100%"}
+
 {% endfigure %}
 {% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Tooltip specs"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Tooltips/tooltip-specs.svg){:width="100%"}
+</div>
+
+## Tooltip variations
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Help text with a link
+{: .hxSubSectionTitle}
+
+Use this variation when a tooltip provides access to a singular external webpage.
+
+- This link should always be an external link and should open in a new tab. 
+- Be sure you always include an external link icon.
+- If you need additional inputs or interactions on a tooltip, for example, a dropdown or selector, use the larger popover pattern.
+
+See [Help links]({{site.baseurl}}/components/help-link.html) for link specifications.
+
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Spacing for tooltip with an associated link"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-text-with-link.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Large tooltips
+{: .hxSubSectionTitle}
+
+Occassionally a tooltip needs to display a robust set of data that falls outside the bounds of traditional text conventions. The larger tooltips follow the same text conventions and sizing guidelines.
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Some tooltips contain larger amounts of information, like definitions and pricing structure. Such tooltips **should be used sparingly**."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-large-tooltips.svg){:width="100%"}
+
 {% endfigure %}
 {% endcolumn %}
 </div>
+
+## Loading state
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Loading states are rarely found when using the tooltip pattern. The loading state is applied exclusively when using data visualizations.
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Loading State."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-loading.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+
+## Interaction behaviors
+{: .hxSectionTitle}
+
+Tooltips can be activated via either a hover or a click on the trigger element.
```

Comment:
Put this line and the header above it in a Helix column. I think we're doing size 8 for stand-alone text, but 4 would also work with it being so short. It will help with later layout for mobile without needing to come back and adjust.

Reply:
done:

```
{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
## Interaction behaviors


Tooltips can be activated via either a hover or a click on the trigger element.
{% endcolumn %}
```